### DISCUSSION TITLE
can I use alpine in NextJs ?

### DISCUSSION THREAD

**Discussion by Neeraj1005** [alpinejs/alpine#1141] [1. Help]
In my laravel project for front-end part I'm using the `Next-js` can you please tell me can I use alpineJs for that.
If yes then How to use ?
Because in my case it send me error and `@click` `x-show` not working.

**Comment by ryangjchandler** [comment]
@Neeraj1005 If you're using Next, is there any reason why you wouldn't want to use React instead of Alpine? @HugoDF wrote an article about using Alpine with React, read it here: https://codewithhugo.com/alpine-js-react/

I'm not sure how much this applies to a Next setup, but it might help.

**Reply by Neeraj1005** [reply]
>is there any reason why you wouldn't want to use React instead of Alpine?

yes basically I'm beginner in Next or React field but I know Alpine Js. Show for little module like dropdown modal I want to use Alpine instead of React. Thats why I'm trying to use alpine in next.

` https://codewithhugo.com/alpine-js-react/` I read this article but not working in my case :(

**Reply by ryangjchandler** [reply]
@Neeraj1005 You might be interested in this issue: https://github.com/alpinejs/alpine/issues/246
